samples/snippets/create_import_job.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+
+# [START kms_create_import_job]
+def create_import_job(project_id, location_id, key_ring_id, import_job_id):
+    """
+    Create a new import job in Cloud KMS.
+    
+    Args:
+        project_id (string): Google Cloud project ID (e.g. 'my-project').
+        location_id (string): Cloud KMS location (e.g. 'us-east1').
+        key_ring_id (string): ID of the Cloud KMS key ring (e.g. 'my-key-ring').
+        import_job_id (string): ID of the import job (e.g. 'my-import-job').
+    Returns:
+        ImportJob: The import job.
+    """
+
+  # Import the client library.
+  from google.cloud import kms
+
+  # Create the client.
+  client = kms.KeyManagementServiceClient()
+
+  # Retrieve the fully-qualified import_job and key_ring string.
+  import_job_name = client.import_job_path(
+      project_id, location_id, key_ring_id, import_job_id)
+  key_ring_name = client.key_ring_path(project_id, location_id, key_ring_id)
+
+  import_job_params = {'import_method': 'RSA_OAEP_4096_SHA1_AES_256', 'protection_level': 'HSM'}
```

Comment:
Improving our samples style guides in an ongoing process, so we generally try to improve new samples going forward rather than update previous versions of the samples.  

In this case, we want to make sure to give users a pointer values of "magic strings". Since there is a limited set of meaningful values here, but they are typed in a way that doesn't clue users to what those values are (strings instead of using an enum), we want to make sure to link to the documentation so users can find out what other values are valid.